From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Markus Hoffrogge <mhoffrogge@gmail.com>
Date: Thu, 10 Nov 2022 13:23:46 +0100
Subject: [ARCHEOLOGY] Fix U-Boot SUNXI nand SPL (#4402)

> X-Git-Archeology: > recovered message: > This fix is provided by mainline U-Boot since v2022.10.
> X-Git-Archeology: > recovered message: > For details refer to:
> X-Git-Archeology: > recovered message: > https://github.com/u-boot/u-boot/commit/5fd30ed78539e11c2c155001a88f483441a96ebd
> X-Git-Archeology: > recovered message: > fixes #4401
> X-Git-Archeology: - Revision 971207799b47d396e6fa6319bd5b5a04aff851f7: https://github.com/armbian/build/commit/971207799b47d396e6fa6319bd5b5a04aff851f7
> X-Git-Archeology:   Date: Thu, 10 Nov 2022 13:23:46 +0100
> X-Git-Archeology:   From: Markus Hoffrogge <mhoffrogge@gmail.com>
> X-Git-Archeology:   Subject: Fix U-Boot SUNXI nand SPL (#4402)
> X-Git-Archeology:
> X-Git-Archeology: - Revision f6a09def35e647a5442e0f92f399485be29f19a9: https://github.com/armbian/build/commit/f6a09def35e647a5442e0f92f399485be29f19a9
> X-Git-Archeology:   Date: Thu, 10 Nov 2022 21:49:36 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Moving patches to per board and removing obsolete (#4409)
> X-Git-Archeology:
---
 drivers/mtd/nand/raw/sunxi_nand_spl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mtd/nand/raw/sunxi_nand_spl.c b/drivers/mtd/nand/raw/sunxi_nand_spl.c
index a29a76c58d..6de0b0a355 100644
--- a/drivers/mtd/nand/raw/sunxi_nand_spl.c
+++ b/drivers/mtd/nand/raw/sunxi_nand_spl.c
@@ -208,7 +208,7 @@ static void nand_apply_config(const struct nfc_config *conf)
 
 	val = readl(SUNXI_NFC_BASE + NFC_CTL);
 	val &= ~NFC_CTL_PAGE_SIZE_MASK;
-	writel(val | NFC_CTL_RAM_METHOD | NFC_CTL_PAGE_SIZE(conf->page_size),
+	writel(val | NFC_CTL_PAGE_SIZE(conf->page_size),
 	       SUNXI_NFC_BASE + NFC_CTL);
 	writel(conf->ecc_size, SUNXI_NFC_BASE + NFC_CNT);
 	writel(conf->page_size, SUNXI_NFC_BASE + NFC_SPARE_AREA);
-- 
Armbian

